Frameworks/UIKit/StarboardXaml/StarboardXaml.h

```diff
@@ -58,7 +58,7 @@ ref class AppEventListener
 
 extern "C" void _ApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg);
 extern "C" void EbrApplicationActivated(Windows::ApplicationModel::Activation::IActivatedEventArgs^ args);
```

Comment:
or would that be a breaking change?

---
In reply to: [72336614](https://github.com/Microsoft/WinObjC/pull/686#discussion_r72336614) [](ancestors = 72336614)